moc_openstack_api_reporting.py

```diff
@@ -0,0 +1,572 @@
+            "id": vol.id,  # "ID of the storage volume",
+            "size": str(vol.size),  # "Size in GB of the storage volume"
+        }
+        event_item["block_devices"].append(volume_data)
+
+    return event_item
+
+
+# 'compute.instance.create.error': 1,
+# 'compute.instance.create.start': 1,
+# 'compute.instance.shutdown.end': 1,
+# 'compute.instance.shutdown.start'
+# {
+#     "disk_gb": "20",
+#     "domain": "Default",
+#     "ephemeral_gb": "0",
+#     "event_type": "compute.instance.create.start",
+#     "generated": "2018-04-18T14:58:38.977823",
+#     "host": "srv-p24-33.cbls.ccr.buffalo.edu",
+#     "instance_id": "703608ed-0b5c-4968-ba93-30f081bf7aec",
+#     "instance_type": "c1.m4",
+#     "instance_type_id": "9",
+#     "memory_mb": "4096",
+#     "message_id": "6e237402-fbae-4af9-bc18-76db88c3706b",
+#     "project_id": "4aeb007a4f9020333a1a1be224bef276",
+#     "project_name": "zealous",
+#     "raw": {},
+#     "request_id": "req-0e8a4b00-b47b-4e68-855d-c13a2bb44032",
+#     "resource_id": "703608ed-0b5c-4968-ba93-30f081bf7aec",
+#     "root_gb": "20",
+#     "service": "compute",
+#     "state": "building",
+#     "tenant_id": "4aeb007a4f9020333a1a1be224bef276",
+#     "user_id": "3abcb51ff942a45b52ac90915ef4c7fb",
+#     "user_name": "yerwa",
+#     "vcpus": "1",
+# }
+
+# compute.instance.create.end': 1,
+# compute.instance.delete.start': 1,
+# compute.instance.live_migration._post.end': 1,
+# compute.instance.live_migration._post.start': 1,
+# compute.instance.live_migration.post.dest.end': 1,
+# compute.instance.live_migration.post.dest.start': 1,
+# compute.instance.live_migration.pre.end': 1,
+# compute.instance.live_migration.pre.start': 1,
+# compute.instance.power_off.end': 1,
+# compute.instance.power_off.start': 1,
+# compute.instance.power_on.end': 1,
+# compute.instance.power_on.start': 1,
+# compute.instance.shutdown.end': 1,
+# compute.instance.shutdown.start
+# {
+#     "disk_gb": "20",
+#     "domain": "Default",
+#     "ephemeral_gb": "0",
+#     "event_type": "compute.instance.create.end",
+#     "generated": "2018-04-18T14:58:45.774847",
+#     "host": "srv-p24-33.cbls.ccr.buffalo.edu",
+#     "instance_id": "703608ed-0b5c-4968-ba93-30f081bf7aec",
+#     "instance_type": "c1.m4",
+#     "instance_type_id": "9",
+#     "launched_at": "2018-04-18T14:58:45.685846",
+#     "memory_mb": "4096",
+#     "message_id": "135a80e4-355f-4561-9c60-e0d2c184e2c3",
+#     "project_id": "4aeb007a4f9020333a1a1be224bef276",
+#     "project_name": "zealous",
+#     "raw": {},
+#     "request_id": "req-0e8a4b00-b47b-4e68-855d-c13a2bb44032",
+#     "resource_id": "703608ed-0b5c-4968-ba93-30f081bf7aec",
+#     "root_gb": "20",
+#     "service": "compute",
+#     "state": "active",
+#     "tenant_id": "4aeb007a4f9020333a1a1be224bef276",
+#     "user_id": "3abcb51ff942a45b52ac90915ef4c7fb",
+#     "user_name": "yerwa",
+#     "vcpus": "1",
+# }
+
+
+def convt_to_ceilometer_event_types(event):
```

Comment:
you might as well name it `convert_to_....`. I don't think saving 2 characters makes it any longer